app/bundles/FormBundle/Translations/en_US/messages.ini

```
@@ -62,6 +62,8 @@ mautic.form.field.form.inputattr="Input attributes"
 mautic.form.field.form.label="Label"
 mautic.form.field.form.labelattr="Label attributes"
 mautic.form.field.form.lead_field="Matching contact field"
+mautic.form.field.form.leadFieldNotOverwrite="Do not overwrite"
+mautic.form.field.form.leadFieldNotOverwrite.tooltip="If yes, contact field value will not be overwritten with the form field value if not empty."
```

I think we could make this clearer for the user

```suggestion
mautic.form.field.form.leadFieldNotOverwrite.tooltip="If set to yes, the contact field value will not be overwritten with the form field value when the contact field already contains data."
```